configs/sst_gpu-X11-enviroment.yaml

```diff
@@ -7,7 +7,6 @@ data:
 
   packages:
     - libXaw
-    - mesa-libGLw
```

Comment:
This config is for both c9s and eln, and we can only remove this from ELN at this point.  Therefore, as previously mentioned, this config will need to be split; iow separate files for c9s and eln, with this package listed only in the c9s config.

Reply:
ah, sorry about that. Should be fixed now!